HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -163,6 +163,7 @@
         'keep *_hltParticleFlow_*_*',
         'keep *_hltParticleNetDiscriminatorsJetTagsAK8_*_*',
         'keep *_hltParticleNetDiscriminatorsJetTags_*_*',
+	'keep *_hltPFDeepFlavour*_*_*',
```

Comment:
Please do NOT edit this file by hand.  
To request updates open a `CMS-HLT` JIRA ticket, clearly explaining why you need this